packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -167,21 +163,18 @@ class DebuggerController extends DisposableController
 
   ValueListenable<String?> get exceptionPauseMode => _exceptionPauseMode;
 
-  final _isolateRef = ValueNotifier<IsolateRef?>(null);
-
-  ValueListenable<IsolateRef?> get isolateRef => _isolateRef;
-
-  bool get isSystemIsolate => isolateRef.value?.isSystemIsolate ?? false;
+  bool get isSystemIsolate =>
+      appState.isolateRef.value?.isSystemIsolate ?? false;
 
   String get _isolateRefId {
-    final id = isolateRef.value?.id;
+    final id = appState.isolateRef.value?.id;
     if (id == null) return '';
     return id;
   }
 
   void _switchToIsolate(IsolateRef? ref) async {
-    _isolateRef.value = ref;
-    _isPaused.value = false;
+    appState.setIsolateRef(ref);
```

Comment:
Dart style says to avoid methods that start with the words get and set.
I would make this an actual setter on the appState object instead.

Reply:
Just `get': https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

Actual setter causes lint complain that wants the setter to be of the same type as getter, that is ValueListenable, while setter takes IsolateRef?